### PR TITLE
dl/add translations

### PR DESCRIPTION
**Pull request by darrikonn**
# Driving License

## What

Add translations to driving license (suuuuuuuuuuper duuuuper fun)
Google Translation to the rescue!

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 1]
```diff
@@ -1,9 +1,319 @@
...
+    description: 'Yes',
+  },
+  no: {
+    id: 'dl.application:shared.no',
+    defaultMessage: 'Nei',
+    description: 'No',
+  },
+  applicationName: {
+    id: 'dl.application:application.name',
+    defaultMessage: 'Ökuskilríki',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:16]: Ökuskírteini?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 2]
```diff
@@ -1,9 +1,319 @@
...
+    description: 'Driving license',
+  },
+  externalDataTitle: {
+    id: 'dl.application:externalData.title',
+    defaultMessage: 'Upplýsingasöfnun og skilyrði',
+    description: 'Information gathering and conditions',
+  },
+  nationalRegistryTitle: {
+    id: 'dl.application:nationalRegistry.title',
+    defaultMessage: 'Persónuuplýsingar úr Þjóðskrá',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:31]: Persónuuppppplýsingar

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 3]
```diff
@@ -1,9 +1,319 @@
...
+  },
+  userProfileInformationTitle: {
+    id: 'pl.application:userprofile.title',
+    defaultMessage: 'Netfang og símanúmer úr þínum stillingum',
+    description: 'Your user profile information',
+  },
+  userProfileInformationSubTitle: {
+    id: 'pl.application:userprofile.subTitle',
+    defaultMessage:
+      'Til þess að auðvelda umsóknarferlið er gott að hafa stillt netfang og símanúmer á mínum síðum',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:49]: `fyllt út` i staðinn fyrir stillt?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 4]
```diff
@@ -1,9 +1,319 @@
...
+    id: 'pl.application:penaltyPoints.title',
+    defaultMessage: 'Punktastaða úr Ökuskírteinaskrá',
+    description: 'Penalty points from Ökuskírteinaskrá',
+  },
+  penaltyPointsSubTitle: {
+    id: 'pl.application:penaltyPoints.subTitle',
+    defaultMessage:
+      'Til þess að tryggja að notandi hafi heimild til þess að sækja um ökuskírteini út frá punktastöðu',
+    description:
+      'In order to ensure the user has permission to apply for driving license',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:63]: `In order to ensure the user's penalty points do not prohibit him from applying for a driving license`

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 5]
```diff
@@ -1,9 +1,319 @@
...
+  },
+  residenceTitle: {
+    id: 'pl.application:residence.title',
+    defaultMessage: 'Búseta',
+    description: 'Recidence',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:68]: Residence

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 6]
```diff
@@ -1,9 +1,319 @@
...
+  },
+  residenceSubTitle: {
+    id: 'pl.application:residence.subtitle',
+    defaultMessage:
+      'Ég hef fasta búsetu hér á landi eins og hún er skilgreind í VIII. viðauka reglugerðar um ökuskírteini eða tel mig fullnægja skilyrðum um búsetu hér á landi til að fá gefið út ökuskírteini.',
+    description:
+      "I've lived in Iceland according to VIII and here by confirm that I meet the conditions to apply for a driving license",
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:75]: vantar ekki viðauka reglugerðar um ökuskírteini eftir VIII?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 7]
```diff
@@ -1,9 +1,319 @@
...
+  },
+  typeFieldCheckbox: {
+    id: 'pl.application:typeField.checkbox',
+    defaultMessage: 'Tegund ökutækja',
+    description: 'Type of vehicle',
+  },
+  typeFieldCar: {
+    id: 'pl.application:typeField.car',
+    defaultMessage: 'Almenn ökuréttindi',
+    description: 'General car license',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:90]: driving license?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 8]
```diff
@@ -1,9 +1,319 @@
...
+      '5. Notar þú að staðaldri læknislyf eða lyfjablöndur sem geta haft áhrif á meðvitund?',
+    description:
+      '5. Do you always use medicines or combinations of medicines that may affect your consciousness?',
+  },
+  healthDeclaration6: {
+    id: 'pl.application:healthDeclaration.6',
+    defaultMessage:
+      '6. Ert þú háð(ur) áfengi, ávana- og/eða fíkniefnum eða misnotar þú geðræn lyf sem verkað gætu á meðvitund?',
+    description:
+      '6. Are you addicted to alcohol, drugs and / or drugs or are you abusing psychotropic drugs that could affect your consciousness?',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:194]: and/or

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 9]
```diff
@@ -1,9 +1,319 @@
...
+  },
+  healthDeclaration7: {
+    id: 'pl.application:healthDeclaration.7',
+    defaultMessage: '7. Notar þú insúlín og/eða töflur við sykursýki?',
+    description: '7. Do you use insulin and / or tablets for diabetes?',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:199]: and/or

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 10]
```diff
@@ -1,9 +1,319 @@
...
+    description: 'City',
+  },
+  overviewTeacher: {
+    id: 'pl.application:overview.teacher',
+    defaultMessage: 'Ökukennari',
+    description: 'Teacher',
+  },
+  overviewBringData: {
+    id: 'pl.application:overview.bringData',
+    defaultMessage: 'Gögn höfð meðferðis til sýslumanns',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:278]: sýslumaður með stóru S?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 11]
```diff
@@ -1,9 +1,319 @@
...
+    description: 'Will bring along data',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:279]: ... to Sýslumaður?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 12]
```diff
@@ -1,9 +1,319 @@
...
+  },
+  overviewBringPhoneData: {
+    id: 'pl.application:overview.bringPhoneData',
+    defaultMessage: 'Ég kem með mynd og rithandarsýni til sýslumanns',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:283]: S?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 13]
```diff
@@ -1,9 +1,319 @@
...
+    description: 'I bring a picture and a handwriting sample to the sheriff',
```
Comment (dabbeg) [inline libs/application/templates/driving-license/src/lib/messages.ts:284]: Sýslumaður

**Review by dabbeg — CHANGES_REQUESTED** [review]
Nice work, I have a few requests for the translations :slightly_smiling_face:

File: libs/application/templates/driving-license/src/lib/messages.ts (hunk 6 above)
Comment (darrikonn) [reply libs/application/templates/driving-license/src/lib/messages.ts:75]: Yeah, probably falls under later refinements. Just wanted to hook up the translations.

**Review by dabbeg — APPROVED** [review]

**Review by jeremybarbet — APPROVED** [review]
LGTM, I'm also working on translations for the ui-core/ui-shell to be move into contentful [here](https://github.com/island-is/island.is/pull/2570), I just been using google translate in the opposite way from you :P

**Pull request merged**